File: api/app/Http/Requests/News/NewsUpdateRequest.php
<?php

namespace App\Http\Requests\News;

use App\Http\Requests\ApiRequest;

class NewsUpdateRequest extends ApiRequest
{
    public function rules(): array
    {
        return [
            'title'     => 'string|min:15|max:125',
            'content'   => 'string|min:50',
            'tags'      => 'array',
            'tags.*'    => 'integer|exists:tags,id',
            //'city'      => 'integer|exists:cities,id',
            //'pictures'  => 'array|min:1',
            //'pictures.*'=> 'required|string|exists:images,hash',
            'cover'     => 'string|exists:images,hash'
        ];
    }
}

File: mobile_apps/RequestsLibrary/Requests/NewsUpdateRequest.cs
using System.Text.Json.Serialization;

namespace RequestsLibrary.Requests;
public class NewsUpdateRequest(string title, string content, int categoryId, string? cover = null)
{
    [JsonPropertyName("title")]       public string  Title      { get; set; } = title;
    [JsonPropertyName("content")]     public string  Content    { get; set; } = content;
    [JsonPropertyName("cover")]       public string? Cover      { get; set; } = cover;
    [JsonPropertyName("category_id")] public int?    CategoryId { get; set; } = categoryId;
}